pymtl3/passes/rtlir/rtype/test/RTLIRDataType_test.py

```diff
@@ -31,6 +31,7 @@ def test_py_list():
   with expected_failure( RTLIRConversionError, 'should be a field of some struct' ):
     rdt.get_rtlir_dtype( [ 1, 2, 3 ] )
 
+# Yanghui : this test no longer passes.
 def test_py_struct_arg_no_default_value():
   class B( object ):
     def __init__( s, foo ):
```

Comment:
@ptpan

Reply:
This branch improves the instantiation of a type and DSL no longer throws the exception this test originally expected. I've updated the test to match the new error message thrown from the RTLIR generation method.